src/canvas-extractor/canvas_extractor/__main__.py

```diff
@@ -40,12 +40,15 @@
 from canvas_extractor.mapping.assignments import map_to_udm_assignments
 from canvas_extractor.mapping.users import map_to_udm_users
 from canvas_extractor.mapping.sections import map_to_udm_sections
+from canvas_extractor.mapping.submissions import map_to_udm_submissions
 from canvas_extractor.csv_generation.write import (
+    SUBMISSION_ROOT_DIRECTORY,
     write_csv,
     write_multi_csv,
     SECTIONS_ROOT_DIRECTORY,
     USERS_ROOT_DIRECTORY,
     ASSIGNMENT_ROOT_DIRECTORY,
+    write_multi_tuple_csv,
```

Comment:
Is there any way to further differentiate the names `write_csv`, `write_multi_csv`, and `write_multi_tuple_csv` so that, when used below, it will be more obvious to the reader why one function was selected over the others?

Reply:
A good idea.  Luis and I talked about using the Google Classroom code for writing data out, so that's where the naming comes from.  When we extract this out, we should improve the naming.

Reply:
Is it okay to leave it for when we extract that logic in a shared library? or it should be done in this story?

Reply:
I think it's ok to wait.  We should do the extract in the upcoming sprint.

Reply:
Please make sure that you write that as an acceptance criteria in an upcoming (or new) ticket.